src/components/InputField/InputField.js

```diff
@@ -276,7 +276,7 @@ InputField.defaultProps = {
   value: null,
   pattern: '',
   required: false,
-  autocomplete: '',
+  autoComplete: '',
```

Comment:
Does the empty string has the same result as not having the autocomplete attribute to the element?

I thought it's not necessary to have a default value if the property is not required.
See how other properties are used for the attributes in the input field above.

Reply:
It does no harm to have as a default prop even if it is not required.

Reply:
that's true, was just wondering whether the empty string might have an effect, but tested and it seems to be fine.